homeassistant/components/skybell/__init__.py

```diff
@@ -46,6 +47,17 @@
 
 async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     """Set up the SkyBell component."""
+    if DOMAIN in config:
+        async_create_issue(
+            hass,
+            DOMAIN,
+            "removed_yaml",
+            breaks_in_ha_version="2022.9.0",
+            is_fixable=False,
+            severity=IssueSeverity.WARNING,
+            translation_key="removed_yaml",
+        )
```

Comment:
- This can be moved further down below, as the config is already guarded for having a domain there.
- If you make this a version with an issue creation that it is "being removed" we can include it in a patch release.

Reply:
I'm not sure about your second bullet point. This should be in 2022.8.1 so users with the integration get to warning now and I'm about to open the PR for removing the yaml for next release.

Reply:
Made suggestions. If we send out the issue "happening" in the patch release, existing users will be notified in 2022.8.1 already.
For 2022.9.0 we can use the removed yaml version.